### PR TITLE
Feature table row disabled and fix checkbox disabled and color

### PR DESCRIPTION
**Pull request by pochretien**

### CODE REVIEW

File: scss/controls/checkboxes.scss [hunk 1]
```diff
@@ -96,7 +96,7 @@ input[type=checkbox].coveo-checkbox {
 
   &:disabled + button {
     cursor: default;
-    opacity: 0.5;
+    opacity: 0.2;
```
Comment (JPLachance) [inline scss/controls/checkboxes.scss]: Are you sure about that value? Seem's weird to me.

**Comment by JPLachance** [comment]
:shipit:

**Comment by GermainBergeron** [comment]
:shipit:

**Comment by wfortin** [comment]
LGTM

**Pull request merged**